modules/k8s/daemonset_test.go

```diff
@@ -0,0 +1,103 @@
+package k8s
```

Comment:
Can you add the relevant build tags like we have in the other test code in this module? Otherwise these tests will fail in our CI environment.